share/p2p/shrexnd/pb/share.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package share.p2p.shrex.nd;
+
+message GetSharesByNamespaceRequest{
+  bytes root_hash = 1;
+  bytes namespace_id = 2;
+}
+
+message GetSharesByNamespaceResponse{
+  StatusCode status = 1;
+  repeated Row rows = 2;
+}
+
+enum StatusCode {
+  INVALID = 0;
+  OK = 1;
+  NOT_FOUND = 2;
+  INTERNAL = 3;
+  REFUSED = 4;
```

Comment:
In what case would we return a `REFUSED`?

Reply:
I am not convinced we need it, but it happens when there is an error getting shares by namespace on the server

Reply:
then that would be INTERNAL

Reply:
That's what REFUSED is for

Reply:
Ahhh, I mixed up those by looking at the diff and see INTERNAL there first

Reply:
got it - lets resolve